src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/DiscoveryEventsHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+
+    /// <summary>
+    /// The Discovery Events Handler.
+    /// Converts the ITestDiscoveryEventsHandler to ITestDiscoveryEventsHandler2
+    /// </summary>
+    public class DiscoveryEventsHandler : ITestDiscoveryEventsHandler2
```

Comment:
We are not using this anywhere. Please take this separately while enabling changes for Translation Layer.